routes/items.js

```diff
@@ -1,10 +1,27 @@
 const express = require('express')
 const router = express.Router()
 
-router.get( '/', ( request, response ) => {
+router.get( '/', ( request, response, next ) => {
   const { user } = request
-  
+
   response.render( 'items/index', { user })
 })
 
+router.post( '/create', ( request, response, next ) => {
+  // console.log( request.session )
+  // const user_id = request.session.passport.user.id
+  const Item = request.app.get( 'models' ).Item
+  const { title, description } = request.body
+  const { id } = request.user
+  const user_id = id
```

Comment:
I would avoid renaming variables like this - if we really want to call this `user_id` in this scope, we should just do this to keep it concise:
```es6
const user_id = request.user.id
```